src/hooks/api/getters/usePlaygroundResults/usePlaygroundResults.ts

```diff
@@ -0,0 +1,54 @@
+import { FeatureSchema } from 'openapi';
+
+export type PlaygroundResultsFeatureSchema = Pick<
+    FeatureSchema,
+    'name' | 'description' | 'project' | 'enabled'
+>;
+
+interface PlaygroundResultsInputSchema {
+    context: object;
+}
+
+export interface IUsePlaygroundResultsOutput {
+    features: PlaygroundResultsFeatureSchema[];
+    input: PlaygroundResultsInputSchema;
+    refetchFeatures: () => void;
+    loading: boolean;
+    error?: Error;
+}
+1;
+
+export const usePlaygroundResults = (): IUsePlaygroundResultsOutput => {
```

Comment:
Nice. We'll just need to coordinate this implementation with @Tymek and @thomasheartman when we put the pieces together.

Reply:
On second thought, this might be uneccesary. This would be an API action that returns the data after we do the request. So the parent component would call the POST endpoint through an API action, receive the response and put the data into state to feed down to this component. We don't need to separately GET this after we do the API call.

Reply:
Yes - agreed, removed when integrating